content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,98 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+This release includes the following new [features](#features) and several [bug fixes](#bug-fixes).
+
+### Features
+
+- [Technical preview of replicating data remotely](#replicate-data-remotely-technical-preview)
+- [Flux updates](#flux-updates)
+- [Build maintenance](#build-maintenance)
+- [Task metadata](#task-metadata)
+- [Metrics](#metrics)
+- [Display runtime configuration in use by `influxd`](#display-runtime-configuration-in-use-by-influxd)
+
+#### Replicate data remotely (technical preview)
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+  In this **technical preview**, you can replicate inbound data using the HTTP API. Support for replicating data via tasks (for example, downsampling) is targeted prior to the official release of this feature.
```

Comment:
Do we have a definition for "technical preview"?

> ...is targeted prior to the official release

What does "is targeted prior" mean here?

Reply:
Good question, @jstirnaman - verifying with @timhallinflux on the official "technical preview" definition. 
Believe the intention is to give folks half of a pie to start eating (replicate inbound data w API), but know they’re going to ask where the other half is (replicate processed data outside of API too), so we’re getting ahead of that…

agree the "is targeted prior" isn't super explicit...for purposes of the release notes, we should prob omit[ future](https://developers.google.com/style/future context) from official docs, and leave that part in release announcements, blogs, marketing teasers, and those channels

Reply:
Removed:

Support for replicating data via tasks (for example, downsampling) is targeted prior to the official release of this feature.